config/settings.yml

```diff
@@ -1,10 +1,10 @@
 stacks:
   storage_root: /stacks
-  djatoka_url: 'http://localhost:8080/resolver'
   driver: djatoka
   djatoka:
     image: DjatokaImage
     info: DjatokaInfoService
+    url: 'http://localhost:8080/resolver'
```

Comment:
This really seems like a property of DjatokaImage.   How would you feel about:
```
   djatoka:
     image:
       class: DjatokaImage
       attributes:
          url: 'http://localhost:8080/resolver'
     info:
       class: DjatokaInfoService

```

Reply:
Then we can have the factory set all the attributes.

Reply:
WFM